### ISSUE TITLE
bug in line number change tracking

### ISSUE THREAD

**Issue by vtjnash** [timholy/Revise.jl#129]
```
julia> using Revise

julia> Revise.track(InteractiveUtils) # lots of changes (additions, deletions of kwarg funcs)

julia> 1
┌ Warning: `Array{T, 1}(m::Int) where T` is deprecated, use `Array{T, 1}(undef, m)` instead.
│   caller = top-level scope at none:0
└ @ Core none:0
ERROR: MethodError: no method matching getindex(::Revise.RelocatableExpr, ::Int64)
Stacktrace:
 [1] eval_revised!(::Revise.FMMaps, ::Module, ::Revise.FMMaps, ::Revise.FMMaps) at /Users/jameson/.julia/packages/Revise/eTozG/src/Revise.jl:177
 [2] eval_revised(::OrderedCollections.OrderedDict{Module,Revise.FMMaps}, ::OrderedCollections.OrderedDict{Module,Revise.FMMaps}) at /Users/jameson/.julia/packages/Revise/eTozG/src/Revise.jl:152
 [3] revise_file_now(::String) at /Users/jameson/.julia/packages/Revise/eTozG/src/Revise.jl:345
 [4] revise(::REPL.REPLBackend) at /Users/jameson/.julia/packages/Revise/eTozG/src/Revise.jl:398
 [5] macro expansion at /Users/jameson/.julia/packages/Revise/eTozG/src/Revise.jl:652 [inlined]
 [6] (::getfield(Revise, Symbol("##41#43")){REPL.REPLBackend})() at ./task.jl:262
```

I think Revise.jl needs a patch something like:
```
+176             if val !== nothing && fmmref.defmap[defref] isa Tuple
```

**Comment by timholy** [comment]
That probably comes from failing to instantiate the signature types. May have been fixed in 5806644d2?

**Comment by vtjnash** [comment]
Actually, I had already installed the `master` version, but just forgot to post that it fixed it. Thanks!
